Makefile

```diff
@@ -32,7 +32,7 @@ ensure::
 	cd tests && go mod tidy
 
 k8sgen::
-	(cd provider && CGO_ENABLED=0 go build -o $(WORKING_DIR)/bin/${CODEGEN} -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" ${PROJECT}/${PROVIDER_PATH}/cmd/$(CODEGEN))
+	(cd provider && CGO_ENABLED=1 go build -o $(WORKING_DIR)/bin/${CODEGEN} -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" ${PROJECT}/${PROVIDER_PATH}/cmd/$(CODEGEN))
```

Comment:
Looks like the github.com/rjeczalik/notify CGO switch is also affecting us here.

Reply:
We will need to update goreleaser here

Reply:
Maybe we pull this out and make the fixes to the build system separately rather than getting stuck behind the 1.24.0 release?